FilePersistence/src/version_control/GitPiecewiseLoader.cpp

```diff
@@ -42,27 +41,36 @@ GitPiecewiseLoader::~GitPiecewiseLoader() {}
 NodeData GitPiecewiseLoader::loadNodeData(Model::NodeIdType id)
 {
 
-	auto regEx = id.toString() + " {.*}";
-	SystemCommandResult result;
-
 	bool isWorkDir = (revision_ == GitRepository::WORKDIR);
-	// in case of workdir just use regular grep
-	// TODO is grep called correctly like this?
+
 	if (isWorkDir)
-		result = runSystemCommand("grep", {"-G", "-r", regEx}, workDir_);
+	{
+		auto regEx = ".*" + id.toString() + " {.*}.*";
+		qDebug() << regEx;
```

Comment:
Please do not commit `qDebug` lines. Using `git gui` you can select individual lines of a modified file to commit, so you don't need to do all or nothing.